File: Football/py3/fb.py
# football soln in py3 for code eval by steven a dunn

import sys, collections

f = open(sys.argv[1], 'r')
for line in f:
	line = line.strip().split(" | ")
	countries = {}
	team_idx = 0
	for team in line:
		fans = list(map(int, team.strip().split()))
		for fan in fans:
			if fan in countries:
				countries[fan].append(team_idx + 1)
			else:
				countries[fan] = [team_idx + 1]
		team_idx += 1

	countries = collections.OrderedDict(sorted(countries.items()))
	for k, v in countries.items():
		v = list(map(str, v))
		sys.stdout.write(str(k) + ":" + ",".join(v) + "; ")
	print()
f.close()

File: Football/c/fb.c
// football soln in c for code eval by steven a dunn

#include <stdlib.h>
#include <stdio.h>
#include <string.h>

int main (int argc, char* const argv[])
{
	FILE *fp = fopen(argv[1], "r");
	if (fp)
	{
		char line[256];
		while(fgets(line, sizeof(line), fp))
		{
			char *token;
			int country_idx;

			country_idx = 0;
			token = strtok(line, "|");
			while (token != NULL)
			{
				
				token = strtok(NULL, "|");
			}

			
		}
		fclose(fp);
	}
	return 0;
}